source/user-manual/user-administration/password-management.rst

```diff
@@ -99,6 +99,7 @@ If you use the tool in an all-in-one deployment, it automatically updates the pa
 
 If you want to change the password for a Wazuh manager API user, run the script on a Wazuh server node and use option ``-A, --api``. Alternatively, you can change the Wazuh manager API passwords following the instructions in the :doc:`Securing the Wazuh API </user-manual/api/securing-api>` documentation.
 
+.. note:: If you want to change the password for Filebeat in the Wazuh server, you don't need to use option ``-A, --api``.
```

Comment:
@Enaraque More useful would be to say what to do rather than what not to do. And I don't see the need to use a note here. Just a paragraph like the previous ones would be ok.